docs/examples/http_exceptions.py

```diff
@@ -0,0 +1,49 @@
+import json
+
+from nameko.web.handlers import HttpRequestHandler
+from werkzeug.wrappers import Response
+
+
+class ApplicationError(Exception):
+    pass
+
+
+class HttpError(Exception):
+
+    def __init__(self, payload, status_code=None):
```

Comment:
such subclasses sound reasonable. and indeed, i would expect those classes to decide on status codes, not the service code raising the exception

in your example, i would expect something closer to `raise HttpError("This is (an?) invalid request")` or whatever (ideally some more detail, e.g. invalid argument foo), and have the class together with `response_from_exception` construct the payload

btw, how come the status code is also included in the payload?